libs/trainer/bbhnet/trainer/trainer.py

```diff
@@ -179,6 +177,13 @@ def train(
     model = architecture(num_ifos)
     model.to(device)
 
+    # if we passed a module for preprocessing,
+    # include it in the model so that the weights
+    # get exported along with everything else
+    if preprocessor is not None:
+        preprocessor.to(device)
+        model = torch.nn.Sequential(preprocessor, model)
+
```

Comment:
So if I understand this correctly, for the StandardScaler, the standardization will be done batch wise, not over the entire training set.

Reply:
Yeah so all these preprocessing modules are meant to be called batch-wise so that they can be built into the model and their parameters get exported with it. Given the fact that some of our data gets created at loading time, it makes less sense to preprocess everything up front